lms/views/basic_lti_launch.py

```diff
@@ -54,8 +54,7 @@ def basic_lti_launch(self, document_url=None, grading_supported=True):
         if grading_supported:
             self.context.js_config.maybe_enable_grading()
 
-        if document_url is not None:
```

Comment:
Was only called with none from `canvas_file_basic_lti_launch`. Not we pass the URL always.